src/context/FoxEthProvider/FoxEthProvider.tsx

```diff
@@ -30,38 +20,25 @@ type FoxEthProviderProps = {
 type IFoxEthContext = {
   farmingAccountId: AccountId | undefined
   setFarmingAccountId: (accountId: AccountId | undefined) => void
-  lpAccountId: AccountId | undefined
-  setLpAccountId: (accountId: AccountId | undefined) => void
+  // TODO(gomes): now that LP is gone, how about we remove this whole hook
```

Comment:
TODO for a follow up, removed all the ETH/FOX LP references - farming will be next, this all can be pure and doesn't need to be a hook anymore